src/index.js

```diff
@@ -35,7 +36,7 @@ export const cn =
       didTwMergeConfigChange = false;
       cachedTwMerge = isEmptyObject(cachedTwMergeConfig)
         ? twMergeBase
-        : extendTailwindMerge(cachedTwMergeConfig);
+        : extendTailwindMerge({extend: cachedTwMergeConfig});
```

Comment:
v2 changes the input format here. This approach maintains the existing API for the config object, but unsure if we should move the format change up the stack.